cmd/zpool/zpool_main.c

```diff
@@ -1107,6 +1107,11 @@ zpool_do_create(int argc, char **argv)
 		}
 	}
 
+	/* Make new pools default to partition=raw */
+	if (add_prop_list_default(zpool_prop_to_name(ZPOOL_PROP_PARTITION),
+	    "raw", &props, B_TRUE))
+		goto errout;
```

Comment:
I don't think I understand how it works. If RAID or whatever mess up the alignment, why would starting at 1MB offset fix it? Would it still aligns to the same place, just 1MB apart?